db/schema.rb

```diff
@@ -176,6 +170,8 @@
     t.datetime "back_image_submit_at", precision: nil
     t.datetime "capture_mobile_back_image_submit_at", precision: nil
     t.datetime "mobile_back_image_submit_at", precision: nil
+    t.integer "link_sent_view_count"
+    t.datetime "link_sent_view_at"
```

Comment:
did this get re-added? it's not in the migration?

Reply:
oh I see now, it got re-ordered? how did that happen, and where did its default/precision go?

Reply:
Not sure but I rebuilt the database from `main`, then applied the new migration and it put it back as it had been so I'm guessing it was due to restoring it by rolling back the migration which didn't include the precision/default. Fixed in 16f53bbb9